tensorflow/compiler/xla/service/gpu/matmul_utils.cc

```diff
@@ -411,7 +412,11 @@ StatusOr<se::blas::ComputationType> GetBlasComputationType(
       return se::blas::ComputationType::kF32;
     case F32:  // fall-through
     case C64:
-      return se::blas::ComputationType::kTF32AsF32;
+      if (tsl::tensor_float_32_execution_enabled()) {
```

Comment:
So what seems to work is this:

```
#if GOOGLE_CUDA
      if (tsl::tensor_float_32_execution_enabled()) {
        return se::blas::ComputationType::kTF32AsF32;
      }
#endif
      return se::blas::ComputationType::kF32;
```

and move the header include above to the #if GOOGLE_CUDA section.

Reply:
Sounds good to me.